packaging/rpm/init.d/grafana-server

```diff
@@ -90,7 +88,7 @@ case "$1" in
 
     # Start Daemon
     cd $GRAFANA_HOME
-    su -s /bin/sh -c "nohup ${DAEMON} ${DAEMON_OPTS} >> /dev/null 3>&1 &" $GRAFANA_USER 2> /dev/null
+    action $"Starting $DESC: ..." su -s /bin/sh -c "nohup ${DAEMON} ${DAEMON_OPTS} >> /dev/null 3>&1 &" $GRAFANA_USER 2> /dev/null
```

Comment:
dammit, my fears where justified :( 
seems action is not available in common distro versions :(

https://github.com/grafana/grafana/issues/6968